### ISSUE TITLE
Duplicate label when exporting

### ISSUE THREAD

**Issue by kynaston** [highcharts/highcharts#5749]
#### Expected behaviour
When sending an SVG to the Exports server, we expect to receive back an accurate (PNG) representation of the SVG.

#### Actual behaviour
The PNG returned has duplicate bar labels for the first bar (the SVG does not).

#### Live demo with steps to reproduce
- Go to http://export.highcharts.com/
- Select "SVG content"
- Input the SVG (see below to this ticket)
- Select image/png
- Input "1200" for the width
- Input "2" for the scale
- Use "chart" as the constructor
- Input nothing for the callback
- Press the "preview" button.

#### Affected browser(s)
All

#### Screenshot
![highcharts-export](https://cloud.githubusercontent.com/assets/1210491/18963095/63d2f1bc-866b-11e6-81b9-2c2455b9bd84.png)

**SVG**:
```
<svg xmlns:xlink="http://www.w3.org/1999/xlink" height="400" width="600" xmlns="http://www.w3.org/2000/svg" style="font-family:'lucida grande', 'lucida sans unicode', arial, helvetica, sans-serif;font-size:12px;" version="1.1"><desc>Created with Highcharts 4.1.5</desc><defs><clipPath id="highcharts-123"><rect height="224" width="518" y="0" x="0"></rect></clipPath></defs><rect class=" highcharts-background" fill="#FFFFFF" strokeWidth="0" height="400" width="600" y="0" x="0"></rect><g  class="highcharts-grid"></g><g  class="highcharts-grid"><path opacity="1"  stroke-width="1" stroke="#D8D8D8" d="M 72 266.5 L 590 266.5" fill="none"></path><path opacity="1"  stroke-width="1" stroke="#D8D8D8" d="M 72 210.5 L 590 210.5" fill="none"></path><path opacity="1"  stroke-width="1" stroke="#D8D8D8" d="M 72 154.5 L 590 154.5" fill="none"></path><path opacity="1"  stroke-width="1" stroke="#D8D8D8" d="M 72 98.5 L 590 98.5" fill="none"></path><path opacity="1"  stroke-width="1" stroke="#D8D8D8" d="M 72 42.5 L 590 42.5" fill="none"></path></g><g  class="highcharts-axis"><path opacity="1" stroke-width="1" stroke="#C0D0E0" d="M 145.5 266 L 145.5 276" fill="none"></path><path opacity="1" stroke-width="1" stroke="#C0D0E0" d="M 219.5 266 L 219.5 276" fill="none"></path><path opacity="1" stroke-width="1" stroke="#C0D0E0" d="M 293.5 266 L 293.5 276" fill="none"></path><path opacity="1" stroke-width="1" stroke="#C0D0E0" d="M 367.5 266 L 367.5 276" fill="none"></path><path opacity="1" stroke-width="1" stroke="#C0D0E0" d="M 441.5 266 L 441.5 276" fill="none"></path><path opacity="1" stroke-width="1" stroke="#C0D0E0" d="M 515.5 266 L 515.5 276" fill="none"></path><path opacity="1" stroke-width="1" stroke="#C0D0E0" d="M 590.5 266 L 590.5 276" fill="none"></path><path opacity="1" stroke-width="1" stroke="#C0D0E0" d="M 71.5 266 L 71.5 276" fill="none"></path><path visibility="visible"  stroke-width="1" stroke="#C0D0E0" d="M 72 266.5 L 590 266.5" fill="none"></path></g><g  class="highcharts-axis"><text y="154" visibility="visible" style="color:#707070;fill:#707070;" class=" highcharts-yaxis-title" transform="translate(0,0) rotate(270 25.149999618530273 154)" text-anchor="middle"  x="25.149999618530273">Percentage</text></g><g  class="highcharts-series-group"><g clip-path="url(#highcharts-123)" transform="translate(72,42) scale(1 1)"  visibility="visible" class="highcharts-series"><rect ry="0" rx="0" fill="rgb(1,125,195)" stroke-width="1" stroke="rgb(255,255,255)" stroke-opacity="1" height="188" width="36" y="36.5" x="18.5"></rect><rect ry="0" rx="0" fill="rgb(102,177,219)" stroke-width="1" stroke="rgb(255,255,255)" stroke-opacity="1" height="29" width="36" y="195.5" x="92.5"></rect><rect ry="0" rx="0" fill="rgb(178,216,237)" stroke-width="1" stroke="rgb(255,255,255)" stroke-opacity="1" height="7" width="36" y="217.5" x="166.5"></rect><rect ry="0" rx="0" fill="rgb(178,178,178)" stroke-width="1" stroke="rgb(255,255,255)" stroke-opacity="1" height="0" width="36" y="224.5" x="240.5"></rect><rect ry="0" rx="0" fill="rgb(222,191,209)" stroke-width="1" stroke="rgb(255,255,255)" stroke-opacity="1" height="0" width="36" y="224.5" x="314.5"></rect><rect ry="0" rx="0" fill="rgb(187,127,163)" stroke-width="1" stroke="rgb(255,255,255)" stroke-opacity="1" height="0" width="36" y="224.5" x="388.5"></rect><rect ry="0" rx="0" fill="rgb(142,41,101)" stroke-width="1" stroke="rgb(255,255,255)" stroke-opacity="1" height="0" width="36" y="224.5" x="462.5"></rect></g><g clip-path="none" transform="translate(72,42) scale(1 1)"  visibility="visible" class="highcharts-markers"></g></g><text y="20" style="color:#333333;font-size:18px;padding:1em 0 0.8em;font-size:1.15em;margin:0;fill:#333333;width:536px;"  class="highcharts-title" text-anchor="start" x="10"><tspan>1) I trust the leaders in my organisation</tspan></text><g opacity="1" transform="translate(72,42) scale(1 1)"  visibility="visible" class="highcharts-data-labels"><text text-anchor="middle" style="color: rgb(0, 0, 0); font-size: 9px; font-weight: bold; fill: rgb(0, 0, 0); text-shadow: 0px 0px 6px rgb(255, 255, 255), 0px 0px 3px rgb(255, 255, 255);" transform="translate(0,0) rotate(315 38.85702260395516 16.5)"  padding="5" r="0" y="16.5" x="38.85702260395516"><tspan>84% (27)</tspan></text><text text-anchor="middle" style="color: rgb(0, 0, 0); font-size: 9px; font-weight: bold; fill: rgb(0, 0, 0); text-shadow: 0px 0px 6px rgb(255, 255, 255), 0px 0px 3px rgb(255, 255, 255);" transform="translate(0,0) rotate(315 112.85702260395516 175.5)"  padding="5" r="0" y="175.5" x="112.85702260395516"><tspan>13% (4)</tspan></text><text text-anchor="middle" style="color: rgb(0, 0, 0); font-size: 9px; font-weight: bold; fill: rgb(0, 0, 0); text-shadow: 0px 0px 6px rgb(255, 255, 255), 0px 0px 3px rgb(255, 255, 255);" transform="translate(0,0) rotate(315 186.85702260395516 197.5)"  padding="5" r="0" y="197.5" x="186.85702260395516"><tspan>3% (1)</tspan></text><text text-anchor="middle" style="color: rgb(0, 0, 0); font-size: 9px; font-weight: bold; fill: rgb(0, 0, 0); text-shadow: 0px 0px 6px rgb(255, 255, 255), 0px 0px 3px rgb(255, 255, 255);" transform="translate(0,0) rotate(315 260.85702260395516 204.5)"  padding="5" r="0" y="204.5" x="260.85702260395516"><tspan>0% (0)</tspan></text><text text-anchor="middle" style="color: rgb(0, 0, 0); font-size: 9px; font-weight: bold; fill: rgb(0, 0, 0); text-shadow: 0px 0px 6px rgb(255, 255, 255), 0px 0px 3px rgb(255, 255, 255);" transform="translate(0,0) rotate(315 334.85702260395516 204.5)"  padding="5" r="0" y="204.5" x="334.85702260395516"><tspan>0% (0)</tspan></text><text text-anchor="middle" style="color: rgb(0, 0, 0); font-size: 9px; font-weight: bold; fill: rgb(0, 0, 0); text-shadow: 0px 0px 6px rgb(255, 255, 255), 0px 0px 3px rgb(255, 255, 255);" transform="translate(0,0) rotate(315 408.85702260395516 204.5)"  padding="5" r="0" y="204.5" x="408.85702260395516"><tspan>0% (0)</tspan></text><text text-anchor="middle" style="color: rgb(0, 0, 0); font-size: 9px; font-weight: bold; fill: rgb(0, 0, 0); text-shadow: 0px 0px 6px rgb(255, 255, 255), 0px 0px 3px rgb(255, 255, 255);" transform="translate(0,0) rotate(315 482.85702260395516 204.5)"  padding="5" r="0" y="204.5" x="482.85702260395516"><tspan>0% (0)</tspan></text></g><g  class="highcharts-axis-labels highcharts-xaxis-labels"><text opacity="1" y="282" transform="translate(0,0) rotate(-45 111.59272486435069 282)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:400px;text-overflow:ellipsis;" x="111.59272486435069"><tspan>Strongly agree</tspan></text><text opacity="1" y="282" transform="translate(0,0) rotate(-45 185.59272486435069 282)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:400px;text-overflow:ellipsis;" x="185.59272486435069"><tspan>Agree</tspan></text><text opacity="1" y="282" transform="translate(0,0) rotate(-45 259.59272486435066 282)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:400px;text-overflow:ellipsis;" x="259.59272486435066"><tspan>Somewhat agree</tspan></text><text opacity="1" y="282" transform="translate(0,0) rotate(-45 333.59272486435066 282)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:400px;text-overflow:ellipsis;" x="333.59272486435066"><tspan>Neither agree nor disagree</tspan></text><text opacity="1" y="282" transform="translate(0,0) rotate(-45 407.59272486435066 282)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:400px;text-overflow:ellipsis;" x="407.59272486435066"><tspan>Somewhat disagree</tspan></text><text opacity="1" y="282" transform="translate(0,0) rotate(-45 481.59272486435066 282)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:400px;text-overflow:ellipsis;" x="481.59272486435066"><tspan>Disagree</tspan></text><text opacity="1" y="282" transform="translate(0,0) rotate(-45 555.5927248643507 282)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:400px;text-overflow:ellipsis;" x="555.5927248643507"><tspan>Strongly disagree</tspan></text></g><g  class="highcharts-axis-labels highcharts-yaxis-labels"><text opacity="1" y="270" transform="translate(0,0)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:188px;text-overflow:clip;" x="57">0</text><text opacity="1" y="46" transform="translate(0,0)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:188px;text-overflow:clip;" x="57">100</text><text opacity="1" y="214" transform="translate(0,0)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:188px;text-overflow:clip;" x="57">25</text><text opacity="1" y="158" transform="translate(0,0)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:188px;text-overflow:clip;" x="57">50</text><text opacity="1" y="102" transform="translate(0,0)" text-anchor="end" style="color:#606060;cursor:default;font-size:11px;fill:#606060;width:188px;text-overflow:clip;" x="57">75</text></g><g transform="translate(0,-9999)" style="cursor:default;padding:0;white-space:nowrap;"  class="highcharts-tooltip"><path transform="translate(1, 1)" stroke-width="5" stroke-opacity="0.049999999999999996" stroke="black"  d="M 3 0 L 13 0 C 16 0 16 0 16 3 L 16 13 C 16 16 16 16 13 16 L 3 16 C 0 16 0 16 0 13 L 0 3 C 0 0 0 0 3 0" fill="none"></path><path transform="translate(1, 1)" stroke-width="3" stroke-opacity="0.09999999999999999" stroke="black"  d="M 3 0 L 13 0 C 16 0 16 0 16 3 L 16 13 C 16 16 16 16 13 16 L 3 16 C 0 16 0 16 0 13 L 0 3 C 0 0 0 0 3 0" fill="none"></path><path transform="translate(1, 1)" stroke-width="1" stroke-opacity="0.15" stroke="black"  d="M 3 0 L 13 0 C 16 0 16 0 16 3 L 16 13 C 16 16 16 16 13 16 L 3 16 C 0 16 0 16 0 13 L 0 3 C 0 0 0 0 3 0" fill="none"></path><path d="M 3 0 L 13 0 C 16 0 16 0 16 3 L 16 13 C 16 16 16 16 13 16 L 3 16 C 0 16 0 16 0 13 L 0 3 C 0 0 0 0 3 0" fill="rgb(249, 249, 249)" fill-opacity=" .85"></path><text transform="translate(0,20)" style="font-size:12px;color:#333333;fill:#333333;"  x="8"></text></g></svg>


```

**Comment by gvaartjes** [comment]
It begins starting to show the duplicate label from width = 950px

**Comment by TorsteinHonsi** [comment]
I tried to reproduce this in jsFiddle, but failed - do you have a live demo?

Also, previous cases like this have been caused by the [dataLabels.style.textShadow](http://api.highcharts.com/highcharts/plotOptions.series.dataLabels.style). Try setting the `textShadow` to `none` and see if that help.

**Comment by kynaston** [comment]
@TorsteinHonsi Thanks for your comment on this.
Here's a working JSFiddle: https://jsfiddle.net/bse0Lf6r/ though I actually don't see the first label *at all* when exporting this to PNG/SVG. On our version (as in the ticket) the SVG is fine but running it through the converter at export.highcharts.com fails. My steps above (including the raw SVG) are reproducible, as gvaartjes has confirmed. On this JSFiddle the SVG is also broken, but on our version the SVG exports (see OP) correctly and it's the PNG convert that fails.

I think the error may not actually be in the Highcharts library, but in the conversion code that powers the export.highcharts.com service?

**Comment by kynaston** [comment]
Note: changing the text-shadow *does* work for us (though it looks the same - to my eye?).

I still believe this is a bug that should be fixed on the conversion code, but this workaround is fine for now (is this workaround in documentation anywhere, do you know?)

**Comment by TorsteinHonsi** [comment]
> changing the text-shadow does work for us (though it looks the same - to my eye?).

Yes, it looks the same. The text shadow in this case is white in order to make the data label stand out on a dark background, but in this case the background is already white.

> I think the error may not actually be in the Highcharts library, but in the conversion code that powers the export.highcharts.com service?

That's correct, it would ultimately be a bug in PhantomJS.

> I actually don't see the first label at all when exporting this to PNG/SVG

If we set the `dataLabel.crop` option to false, the label is preserved: https://jsfiddle.net/highcharts/bse0Lf6r/1/. But I can't see the shadow artefact at all in the jsFiddle. Tested with Highcharts 5.0.0 and Highcharts 4.2.7. Are you sure you've tested with the latest version?
